packages/core/content-manager/server/src/history/content-types/history-version.ts

```diff
@@ -0,0 +1,66 @@
+import type { Plugin } from '@strapi/types';
+
+const historyVersion: Plugin.LoadedPlugin['contentTypes'][string] = {
+  schema: {
+    uid: 'plugin::content-manager.history-version',
+    modelName: 'history-version',
+    globalId: 'ContentManagerHistoryVersion',
+    kind: 'collectionType',
+    modelType: 'contentType',
+    collectionName: 'strapi_history_versions',
+    info: {
+      singularName: 'history-version',
+      pluralName: 'history-versions',
+      displayName: 'History Version',
+    },
+    options: {
+      reviewWorkflows: false,
+    },
+    pluginOptions: {
+      'content-manager': {
+        visible: false,
+      },
+      'content-type-builder': {
+        visible: false,
+      },
+      i18n: {
+        localized: true,
+      },
```

Comment:
does the content type need to be localized? I see the locale attribute created below, so I think enabling i18n is not necessary

Reply:
oops, I meant to make it false